pages/Hypr Ecosystem/hyprpaper.md

```diff
@@ -74,3 +96,6 @@ hyprctl hyprpaper listloaded
 ```
 
 Please note all paths have to be absolute (or start with `~`).
+
+## Docs
```

Comment:
I couldn't decide on the title:

- Docs
- Documentation
- Source Code

Reply:
whats the point of this section?